_dashboards/index.md

```diff
@@ -11,15 +11,55 @@ redirect_from:
+<img src="{{site.url}}{{site.baseurl}}/images/dashboard-flight.png" alt="User interface showing several data visualizations">
 
-1. After starting OpenSearch Dashboards, you can access it at port 5601. For example, http://localhost:5601.
-1. Log in with the default username `admin` and password `admin`.
-1. Choose **Try our sample data** and add the sample flight data.
-1. Choose **Discover** and search for a few flights.
-1. Choose **Dashboard**, **[Flights] Global Flight Dashboard**, and wait for the dashboard to load.
+# Use cases for OpenSearch Dashboards
+
+In OpenSearch Dashboards, a data dashboard is a collection of charts, graphs, and insights that give you a snapshot of the data that you're interested in and that you interact with. 
+You can track, analyze, and display real-time search, monitoring, and analysis of business and operational data for use cases like application monitoring, log analytics, observability, and website search. 
+
+##### Example: Data visualization use cases   
+
+<table style="table-layout: fixed; width: 100%;">
+<tbody>
+<tr>
+<td style="text-align: left; font-family:Open Sans Condensed; vertical-align: top; width: 100%;">Visualize log and trace data with interactive log analytics.<img src="{{site.url}}{{site.baseurl}}/images/visualize-log-data.png" alt="Log analytics" /></td>
+<td style="text-align: left; font-family:Open Sans Condensed; width: 100%;">Detect and mitigate issues faster with anomaly detection.<img src="{{site.url}}{{site.baseurl}}/images/anomaly-detection.png" alt="Anomaly detection" /></td>
+</tr>
+<tr>
+<td style="text-align: left; font-family:Open Sans Condensed; vertical-align: top; width: 100%;">Diagnose performance issues and reduce application downtime.<img src="{{site.url}}{{site.baseurl}}/images/observability.png" alt="Observability interface and log monitoring features" /></td>
+<td style="text-align: left; font-family:Open Sans Condensed; vertical-align: top; width: 100%;">Unlock real-time search, monitoring, and analysis of business and operational data.<img src="{{site.url}}{{site.baseurl}}/images/analyzing-data-logs.png" alt="Real-time search, monitoring, and analysis" /></td>
+</tr>
+</tbody>
+</table> 
+
+# Explore the OpenSearch Dashboards playground
+
+You can interact with the demonstration datasets in the OpenSearch Dashboards playground by following the steps below:
+
+1. Go to the [OpenSearch Dashboards playground](https://playground.opensearch.org/app/home).
+2. Choose **OpenSearch Dashboards > Dashboard**. 
+3. Explore the available datasets and select one: **[Flights] Global Flight Dashboard**, **[Logs] Web Traffic**, or **[eCommerce] Revenue Dashboard**.
+4. Interact with the data on a dashboard. Choose **Add filter**, and specify the data you'd like to see. 
+5. Select one of the options from the **Field** menu, and specify an **Operator** to filter the results.  
+
+##### Example: [Logs] Web Traffic dashboard with filters applied
```

Comment:
I would bold this instead of making it it's own header

```suggestion
**Example: [Logs] Web Traffic dashboard with filters applied**
```

Reply:
Done